README.md

```diff
@@ -8,11 +8,11 @@ This is the GitHub page for the Detect plugin for Azure DevOps.
 
 
 ## Where can I get the latest release? ##
-Detect for Azure DevOps is available on the [Azure Marketplace](https://marketplace.visualstudio.com/items?itemName=synopsys-detect.synopsys-detect).
+Detect for Azure DevOps is available on the [Azure Marketplace](https://marketplace.visualstudio.com/items?itemName=blackduck-detect.blackduck-detect).
 
 Releases and Source are available under [GitHub releases](https://github.com/blackducksoftware/detect-ado/releases).
 
 ## Documentation ##
-Instructions and examples are available on our [Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Synopsys+Detect+Extension+for+Azure+DevOps).
+Instructions and examples are available on our [Public Confluence](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Black+Duck+Detect+Extension+for+Azure+DevOps).
```

Comment:
Correct URL appears to be: https://documentation.blackduck.com/bundle/detect/page/integrations/azureplugin/azure.html